src/web/components/App.tsx

```diff
@@ -38,6 +40,16 @@ export const App = ({ CAPI, NAV }: Props) => {
         setIsSignedIn(!!getCookie('GU_U'));
     }, []);
 
+    useEffect(() => {
+        const callGetUser = async () => {
+            setUser(await getUser(CAPI.config.discussionApiUrl));
```

Comment:
is it ok to have `user` as a promise?

Reply:
I don't think that will happen. All being well, the await should mean the promise resolves to the result of the fetch